.github/workflows/build-docs.yml

```diff
@@ -5,15 +5,19 @@ on:
   workflow_call:
     inputs:
         python-version:
+          description: 'used Python version'
           required: true
           type: string
         package-manager:
+          description: 'pip or poetry'
```

Comment:
A tiny note: choose one
1. all lower-case names (i.e. `python`, `pip`, and `poetry`)
2. all capitalized names (i.e. `Python`, `Pip`, and `Poetry`)
3. use the original case (i.e. [`python`](https://www.python.org/), [`pip`](https://pip.pypa.io/), and [`Poetry`](https://python-poetry.org/))

I don't like 2., but the others are both fine (3. is nicer, but more cumbersome to follow consistently, so the two things compensate).

Reply:
Python is also cumbersome on its own: they always capitalize it in text, but the logo itself is lowercase...

Reply:
I prefer the first one, because the inputs are all lower-case name